src/CourseDashboard.cpp

```
@@ -51,34 +54,88 @@ void CourseDashboard::updateUser(User & user)
     std::cin >> number;
     std::string name, nick, group, gitHub, firecode;
-    switch (number) {
-    case 1:
-        std::cout << "Give new value to data: ";
-        std::cin >> name;
-        user.setName(name);
-        break;
-    case 2:
-        std::cout << "Give new value to data: ";
-        std::cin >> nick;
-        user.setNick(nick);
-        break;
-    case 3:
-        std::cout << "Give new value to data: ";
-        std::cin >> group;
-        user.setGroup(group);
-        break;
-    case 4:
-        std::cout << "Give new value to data: ";
-        std::cin >> gitHub;
-        user.setGitHub(gitHub);
-        break;
-    case 5:
-        std::cout << "Give new value to data: ";
-        std::cin >> firecode;
-        user.setFirecode(firecode);
-        break;
-    default:
-        std::cout << "You gave wrong value.";
-        break;
+    switch (number)
+    {
+        case 1:
+            std::cout << "Give new value to data: ";
+            std::cin >> name;
+            user.setName(name);
+            break;
+        case 2:
+            std::cout << "Give new value to data: ";
+            std::cin >> nick;
+            user.setNick(nick);
+            break;
+        case 3:
+            std::cout << "Give new value to data: ";
+            std::cin >> group;
+            user.setGroup(group);
+            break;
+        case 4:
+            std::cout << "Give new value to data: ";
+            std::cin >> gitHub;
+            user.setGitHub(gitHub);
+            break;
+        case 5:
+            std::cout << "Give new value to data: ";
+            std::cin >> firecode;
+            user.setFirecode(firecode);
+            break;
+        default:
+            std::cout << "You gave wrong value.";
+            break;
+    }
+}
+
+void CourseDashboard::saveToFile()
+{
+    std::fstream file_DB("baza danych.txt", file_DB.in | file_DB.out);
+    if(!file_DB.is_open())
+    {
+        std::cout << "error" << std::endl;
```

how to test if a file was not open? This implementation is difficult to test. It's better to throw an exception.

---

changed to throw

---

Problem! Found that was not changed in the commits and local copy - probably my mistake during marking as resolved - will be fixed